src/modules/auto_theater_mode/index.js

```diff
@@ -22,7 +22,9 @@ class AutoTheaterModeModule {
             const container = App.__container__;
             const routeName = container.lookup('controller:application').get('currentRouteName');
             if (routeName !== 'channel.index.index' && ['videos', 'vod'].includes(routeName)) return;
-            if (container.lookup('service:persistentPlayer').playerComponent.player.theatre) return;
+
+            const playerComponent = container.lookup('service:persistentPlayer').playerComponent;
```

Comment:
i feel like this is the wrong fix.. why is channel loading on chat.. that should be prevented tbh